modules/gapi/test/streaming/gapi_streaming_vpl_core_test.cpp

```diff
@@ -0,0 +1,190 @@
+// This file is part of OpenCV project.
```

Comment:
I doubt if these tests test the public aspects of the API - maybe it is worth moving to some `internal` or `vpl` subdirectory.